virtool/subtractions/api.py

```diff
@@ -341,7 +335,7 @@ async def finalize_subtraction(req: aiohttp.web.Request):
         "$set": {
             "gc": data["gc"],
             "ready": True,
-            "files": await virtool.subtractions.utils.prepare_files_field(pg, subtraction_id)
+            "files": await virtool.subtractions.utils.get_subtraction_files(pg, subtraction_id)
```

Comment:
In the story, I meant that the `files` field does not have to exist at all. We can leave all subtraction file data in SQL instead of storing anything in Mongo.